daemon/docker.go

```diff
@@ -83,11 +83,19 @@ func deploy(repo *git.Repository, cli *docker.Client, out io.Writer) error {
 	// See https://cloud.google.com/community/tutorials/docker-compose-on-container-optimized-os
 	fmt.Fprintln(out, "Building project...")
 	ctx := context.Background()
+
+	repoName, err := common.GetProjectName(repo)
+	if err != nil {
+		fmt.Println(repoName)
```

Comment:
Not sure if this is just for work in progress print testing but when printing in the `daemon`, notice that almost every function now has an `io.Writer` parameter. `io.Writer` is an interface that is taken as a parameter by many of Go's functions to point towards a particular output. Interfaces in Go are implicitly inherited so, in this case, any class that implements a function `Write()` is an `io.Writer`!  😮 

On line 84 of this file:
```go
// print to output "out"
fmt.Fprintln(out, "Building project...")
```

Reply:
Oh, sorry! This was from debugging, I'll remove it.

Reply:
Still a useful trick - if you print to `out`, when you test it out you can just use `inertia local logs --stream` and the output will be sent to you 😄

Reply:
Done.